HOW_WE_USE_GITHUB.md

```diff
@@ -265,6 +267,82 @@ Community support can be found elsewhere, though, and we encourage you to explor
 
 In order to not have to manually type or copy/paste the above repeatedly, note that it's possible to add text for the most commonly-used responses via [GitHub's "Add Saved Reply" option][docs-saved-reply].
 
+## Development Processes
+
+The following are best practices the conda organization encourages for feature
+development. While we recommend projects under the conda organization adopt
+these practices, they are not strictly required.
+
+### How should we approach feature development?
+
+For new features, begin by creating an issue for the feature if one doesn't
+already exist. Once a new feature has been added to the backlog, next create a
+specification for the new feature within the project so that you can get early
+input on the design. Specification includes design mockups, API or command
+reference docs, recording the planned approach in the feature issue, and
+creating CLI argument demos without the functionality of the feature.
```

Comment:
Let's make it a little simpler. 

```suggestion
For new features, first open an issue if one doesn’t exist. After it’s added to the backlog, create a specification to gather early feedback. This can include mockups, API/command references, a written plan in the issue, and sample CLI arguments (without functionality).
```

Reply:
Hi @ForgottenProgramme, thanks, I incorporated this suggestion!